src/Type.ts

```diff
@@ -115,3 +117,20 @@ export const lax = <P extends t.Props>(props: P, name?: string) =>
         partial.encode,
       ),
   )
+
+export const decode =
+  <C extends t.Mixed>(codec: C) =>
+  (u: unknown): E.Either<Error, t.TypeOf<C>> =>
```

Comment:
@davidecaruso @pierroberto 

From now on, I suggest using this function to decode via `io-ts` codecs.

Reply:
ok!